docs/user/openstack/customization.md

```diff
@@ -61,6 +61,8 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 > **Note**
 > Note when deploying with `Kuryr` there is an Octavia API loadbalancer VM that will not fulfill the Availability Zones restrictions due to Octavia lack of support for it. In addition, if Octavia only has the amphora provider instead of also the OVN-Octavia provider, all the OpenShift services will be backed up by Octavia Load Balancer VMs which will not fulfill the Availability Zone restrictions either.
 
+> **Note**
```

Comment:
This syntax is deprecated and you should use `[!NOTE]` now.
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
yeah but all the file is like this. I think we should update all of the occurrences in once.